src/api/routers/participants/participantsUsers.ts

```diff
@@ -4,26 +4,20 @@ import { z } from 'zod';
 import { AuditAction, AuditTrailEvents } from '../../entities/AuditTrail';
 import { UserJobFunction } from '../../entities/User';
 import { getTraceId } from '../../helpers/loggingHelpers';
-import { getKcAdminClient } from '../../keycloakAdminClient';
 import {
   constructAuditTrailObject,
   performAsyncOperationWithAuditTrail,
 } from '../../services/auditTrailService';
-import { createNewUser, sendInviteEmailToNewUser } from '../../services/kcUsersService';
 import { ParticipantRequest, UserParticipantRequest } from '../../services/participantsService';
-import {
-  createUserInPortal,
-  findUserByEmail,
-  getAllUserFromParticipant,
-} from '../../services/usersService';
+import { getAllUserFromParticipant, inviteUserToParticipant } from '../../services/usersService';
 
 export async function getParticipantUsers(req: ParticipantRequest, res: Response) {
   const { participant } = req;
   const users = await getAllUserFromParticipant(participant!);
   return res.status(200).json(users);
 }
 
-const invitationParser = z.object({
+export const UserInvitationParser = z.object({
   firstName: z.string(),
   lastName: z.string(),
   email: z.string(),
```

Comment:
nit: Is the convention normally that the first word is capitalized? I see it changed from the first letter being lowercase to upper case

Reply:
good point, I'd say most of them are actually camelCase: 
![image](https://github.com/user-attachments/assets/b36ae297-e01a-4dae-b9df-f2c15d25eabb)

Looking at the [zod docs](https://zod.dev/), it looks like they use PascalCase for [objects](https://zod.dev/?id=objects) and camelCase for [primitives](https://zod.dev/?id=basic-usage). 

ChatGPT argues that PascalCase should be used for models or types that are intended to be reused across the application, while camelCase should be used for schemas that are defined within a specific, narrow scope, like a single function. I tend to agree with that.

Thinking about this more, it probably doesn't make the most sense to include `Parser` in the name either, since it's just a `ZodObject` which has a `parse` method on it.

I've actioned this for this instance - I'm happy to apply that convention more broadly in a separate PR /ticket if we think it's the right move.

thanks!

Reply:
Definitely not urgent, but we could make a separate continuous improvement, quick win ticket for changing this at some point